dockerfiles/alpine/analytics-dashboard/README.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile for WSO2 Micro Integrator #
```

Comment:
@ThilinaManamgoda let's mark this as `Micro Integrator Dashboard` or any suitable name. Otherwise, one may confuse with Micro Integrator runtime.